test/functional-portable/ucp/cloud/aws_credential_test.go

```diff
@@ -26,18 +26,19 @@ import (
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
 	ucp "github.com/radius-project/radius/pkg/ucp/api/v20231001preview"
+	ucptest "github.com/radius-project/radius/test/functional-portable/ucp"
 	"github.com/stretchr/testify/require"
 )
 
 func Test_AWS_Credential_Operations(t *testing.T) {
-	test := NewUCPTest(t, "Test_AWS_Credential_Operations", func(t *testing.T, url string, roundTripper http.RoundTripper) {
+	test := ucptest.NewUCPTest(t, "Test_AWS_Credential_Operations", func(t *testing.T, url string, roundTripper http.RoundTripper) {
```

Comment:
nit: we can also simplify the name. https://sben.dev/stutter/

Up to you if you want to handle this separately.